tools/devops/automation/build-pipeline.yml

```diff
@@ -170,15 +170,19 @@ stages:
     vsdropsPrefix: ${{ variables.vsdropsPrefix }}
     keyringPass: $(xma-password)
 
-- template: templates/mac-tests.yml
+- template: templates/mac/stage.yml
   parameters:
-    stageName: 'Mac Mojave Tests'
+    stageName: 'Mac Mojave (10.14)'
     macPool: 'Hosted Mac Internal Mojave'
+    statusContext: 'Mac Mojave (10.14)'
+    keyringPass: $(xma-password)
 
-- template: templates/mac-tests.yml
+- template: templates/mac/stage.yml
   parameters:
-    stageName: 'Mac High Sierra Tests'
+    stageName: 'Mac High Sierra (10.13)'
```

Comment:
What about macOS 10.9 -> 10.12?

Reply:
Easy to add, I need pools, I'll ping vseng